README.md

```diff
@@ -12,14 +12,28 @@ We recommend using VSCode with the [Vala plugin](https://marketplace.visualstudi
 
 - Alpine Linux Edge: `apk add vala-language-server`
 
-- Ubuntu 18.04 and elementary:
+- Ubuntu 18.04 and elementaryOS
+
+
+  This project requires vala >= 0.48, which is unavailable on those
```

Comment:
Change this entire paragraph to:

> In order to install VLS, you need Vala 0.48 or later, which may be unavailable on your distribution. You can get an up-to-date version of Vala from the Vala Team's Vala-Next repository. First you need to remove older Vala components:

The command list that follows doesn't need to be changed.